pixeltable/functions/timestamp.py

```diff
@@ -17,7 +17,7 @@
 from pixeltable.utils.code import local_public_names
 
 
-@func.udf
+@func.udf(is_method=True)
```

Comment:
This (and month, day, etc.) are properties for Python's datetime, let's make them properties here as well.